scss/devices/brasil/_desktop.scss

```diff
@@ -0,0 +1,153 @@
+@media (min-width: $desktopVersion){
+    // NAV START
+    details {
+        display: none;
+    }
+    nav {
+        height: 10vh;
+        padding: 0 3%;
+        align-items: center;
+        position: sticky;
+        top: 0;
+
+        > ul {
+            @include flexRow(1vw);
+            width: 45%;
+            justify-content: space-around;
+            text-transform: uppercase;
+        }
+
+        aside {
+            padding: 0;
+            width: 10vh;
+            height: 10vh;
+            @include flexRow;
+            justify-content: center;
+            align-items: center;
+        }
+
+        a {
+            color: black;
+            font-weight: 600;
+            &:hover {
+                color: $nav;
+            }
+        }
+    }
+    // NAV END
+    // HEADER START
+    .kHeader {
```

Comment:
I think you should write bHeader. Isnt it?

Reply:
actually, as it is a class, its name doesn't really matter, it will end up working anyway